main/management/commands/kafka_consumer.py

```diff
@@ -23,37 +23,49 @@ def add_arguments(self, parser: ArgumentParser) -> None:
 
     def handle(self, debug: bool = False, **kwargs: Any) -> None:  # type: ignore[misc]
         """Command business logic."""
+        topic_pm = "control.*.process_manager"
+        topic_ers = "erskafka-reporting"
         consumer = KafkaConsumer(bootstrap_servers=[settings.KAFKA_ADDRESS])
-        consumer.subscribe(pattern="control.*.process_manager")
+        consumer.subscribe(pattern=f"^({topic_pm}|{topic_ers})$")
         # TODO: determine why the below doesn't work
         # consumer.subscribe(pattern="control.no_session.process_manager")
 
         self.stdout.write("Listening for messages from Kafka.")
         while True:
             for messages in consumer.poll(timeout_ms=500).values():
-                message_timestamps = []
-                message_bodies = []
+                message_records = []
+
+                # HACK: for testing ERS messages. Remove when ERS endpoint is live.
+                message_records_ers = []
+
```

Comment:
Not really clear on how this enables testing. I was thinking that we could have a script (similar to `scripts/talk_to_process_manager.py`) that would create some messages with the ers topic to facilitate interactive testing of messages in the controller view.

Reply:
I've made a generic `store_message` command, with user-supplied message and topic.